### ISSUE TITLE
Support key mapping

### ISSUE THREAD

**Issue by unional** [ranelpadon/ngram-type#36]
It would be great if the app can support key mapping, so that it can simulate colemak-dh on a qwerty keyboard.

**Comment by ranelpadon** [comment]
Hmm I use Colemak DH also on Mac, and I set the layout in the system's keyboard preferences which is the usual approach. Do you mean you want to practice Colemak DH without changing your keyboard layout/preferences and will let the website/web browser do the re-mapping for you?

**Comment by unional** [comment]
Yes. monkeytype and keybr both support this feature. Its actually a keyboard emulation.

It would be good for people who doesn't really have the alternative layouts be able to try and learn them.
